modules/openapi-generator/src/main/resources/Java/libraries/feign/api_test.mustache

```diff
@@ -1,10 +1,11 @@
 package {{package}};
+//TODO update packages 
```

Comment:
@hugo-ma-alves last question from me. Should this line be removed from the template?

Reply:
Yes, removed